README.md

```diff
@@ -77,10 +103,44 @@ If there is anything wrong with this repository (documentation needed / broken t
 
 ## VS Code Extensions
 
-- _This is completely optional, just if you're curious as to which I'm using_
+_This is completely optional, just if you're curious as to which I'm using_
 
 - In Bash:
   `cat vs-code-extensions-list.txt | xargs -L 1 code --install-extension`
 
 - In PowerShell:
   `Get-Content vs-code-extensions-list.txt | ForEach-Object { code --install-extension $\_ }`
+
+## Troubleshooting Guide
+
+###  Does VirtualBox work in M1 Macs?
+
+- It has been found that VirtualBox does not operate correctly in Mac PCs with an M1 chip. For more information please visit this [Reddit Thread](https://www.reddit.com/r/mac/comments/14rics9/does_virtualbox_work_in_m1_macs/).
+
+### Error: You need to fork this repository
+
+- If an error occurs in GitHub or in your code editor such as "*Sorry, you're not able to edit this repository directly -- You need to fork it and propose changes from there instead*", please ask access to the repository by sending a direct message via Discord to the repository owner. Once access has been granted by the owner, an email will be sent with an invitation to email linked to your GitHub account. Please click the "Accept Invitation" button on the email. This should resolve the issue.
+
+### Vagrant is not working
+
+- If you just installed Vagrant, the device on which it was installed will need to be restarted before the software is usable.
+- Otherwise, please repair vagrant by opening the installer and clicking "Next" > "Repair".
+
+### Vagrant up is taking forever
+
+- The time that the vagrant takes to build the box is heavily dependent on your machine's hardware. It can definitely take much longer than expected.
```

Comment:
The box is "built" already, its more dependent on your Storage (i.e. HD vs SSD r/w) and internet download speeds.  The Box file that gets download with `vagrant up` is approx 10 Gb

Reply:
changed to load